### PR TITLE
BUILDBASE-1546 Rework current Buildbase docs to functional docs repo

### PR DESCRIPTION
**Pull request by strouwi-dds**

### CODE REVIEW

File: Writerside/topics/Client.md [hunk 1]
```diff
@@ -1,72 +0,0 @@
-# Clients
```
Comment (XanderStrouwi) [inline Writerside/topics/Client.md:1]: Why did you remove this one?

File: Writerside/topics/Configuration.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+absence. When committed the absences are only shown on working days, days when work hours are expected.
+
+When choosing a day in the past a message pops up. The user can then choose a new and valid date or add an absence only
+for the selected day
+
+### Prickings
+
+Prickings can be accessed through the navigation bar on the left of the screen. The button will always display a clock symbol, but its text may vary.
+
+![Alt text](/Resources/pricking_timer_no_existing.png)
```
Comment (XanderStrouwi) [inline Writerside/topics/Configuration.md]: The images in this md start at path "Resources". I don't think that's exists here

**Review by XanderStrouwi — CHANGES_REQUESTED** [review]

File: Writerside/topics/Client.md (hunk 1 above)
Comment (strouwi-dds) [reply Writerside/topics/Client.md:1]: It only had some parts that are already mentioned in Clients and projects

**Review by XanderStrouwi — APPROVED** [review]
